src/GitHub.InlineReviews/PullRequestFilterProvider.cs

```diff
@@ -93,7 +93,7 @@ private bool ShouldIncludeInFilter(IVsHierarchyItem hierarchyItem)
 
             private static string BuildAbsolutePath(string solutionDirectory, string fileRelativePath)
             {
-                return Path.Combine(solutionDirectory, fileRelativePath.Replace("/", @"\")).ToLower();
+                return Path.Combine(solutionDirectory, fileRelativePath.Replace("/", @"\")).ToUpperInvariant();
```

Comment:
You introduced a 🐛 there! We need to have it lowercase! I will fix it in a later commit!